system-variables.md

```diff
@@ -1,129 +1,836 @@
+
+- Scope: GLOBAL
+- Default value: 512
+- This variable is used to set the number of retries when the DDL operation fails. When the number of retries exceeds the parameter value, the wrong DDL operation is canceled.
+
+### tidb_ddl_reorg_batch_size
+
+- Scope: GLOBAL
+- Default value: 256
+- This variable is used to set the batch size during the `re-organize` phase of the DDL operation. For example, when TiDB executes the `ADD INDEX` operation, the index data needs to backfilled by `tidb_ddl_reorg_worker_cnt` (the number) concurrent workers. Each worker backfills the index data in batches.
+    - If many updating operations such as `UPDATE` and `REPLACE` exist during the `ADD INDEX` operation, a larger batch size indicates a larger probability of transaction conflicts. In this case, you need to adjust the batch size to a smaller value. The minimum value is 32.
+    - If the transaction conflict does not exist, you can set the batch size to a large value. The maximum value is 10240. This can increase the speed of the backfilling data, but the write pressure on TiKV also becomes higher.
+
+### tidb_ddl_reorg_priority
+
+- Scope: SESSION | GLOBAL
+- Default value: `PRIORITY_LOW`
+- This variable is used to set the priority of executing the `ADD INDEX` operation in the `re-organize` phase.
+- You can set the value of this variable to `PRIORITY_LOW`, `PRIORITY_NORMAL` or `PRIORITY_HIGH`.
+
+### tidb_ddl_reorg_worker_cnt
+
+- Scope: GLOBAL
+- Default value: 4
+- This variable is used to set the concurrency of the DDL operation in the `re-organize` phase.
+
+### tidb_disable_txn_auto_retry
+
+- Scope: SESSION | GLOBAL
+- Default: on
+- This variable is used to set whether to disable the automatic retry of explicit transactions. The default value of `on` means that transactions will not automatically retry in TiDB and `COMMIT` statements might return errors that need to be handled in the application layer.
+
+    Setting the value to `off` means that TiDB will automatically retry transactions, resulting in fewer errors from `COMMIT` statements. Be careful when making this change, because it might result in lost updates.
+
+    This variable does not affect automatically committed implicit transactions and internally executed transactions in TiDB. The maximum retry count of these transactions is determined by the value of `tidb_retry_limit`.
+
+    For more details, see [limits of retry](/optimistic-transaction.md#limits-of-retry).
+
+### tidb_distsql_scan_concurrency
+
+- Scope: SESSION | GLOBAL
+- Default value: 15
+- This variable is used to set the concurrency of the `scan` operation.
+- Use a bigger value in OLAP scenarios, and a smaller value in OLTP scenarios.
+- For OLAP scenarios, the maximum value cannot exceed the number of CPU cores of all the TiKV nodes.
+
+### tidb_enable_cascades_planner
+
+- Scope: SESSION | GLOBAL
+- Default value: 0
+- This variable is used to control whether to enable the cascades planner feature.
```

Comment:
We need to explain more about this variable or not introduce this variable to the user now. Because the cascades planner is under development, it has not been fully tested, lots of functionalities are not supported now.

Reply:
I have explained it is experimental.